pkg/cache/internal/informers_map.go

```diff
@@ -76,15 +70,10 @@ type MapEntry struct {
 // specificInformersMap create and caches Informers for (runtime.Object, schema.GroupVersionKind) pairs.
 // It uses a standard parameter codec constructed based on the given generated Scheme.
 type specificInformersMap struct {
-	// Scheme maps runtime.Objects to GroupVersionKinds
-	Scheme *runtime.Scheme
```

Comment:
This change seems outside the scope of this PR?

Reply:
Restored